testing.py

```diff
@@ -381,6 +381,8 @@ def _lane(self, display_name=None, library=None,
             parent=parent, display_name=display_name,
             create_method_kwargs=dict(fiction=fiction)
         )
+        if is_new and parent:
+            lane.priority = len(parent.sublanes)-1
```

Comment:
This guarantees that lanes act the way you would expect in a test -- their priority is the order they were created.